server/src/main/resources/config/feeds.conf

```diff
@@ -23,6 +23,8 @@ acl {
   adjustment.number-of-days-in-average = 3
   adjustment.number-of-days-in-average = ${?ACL_ADJUSTMENT_NUMBER_OF_DAYS}
   min-file-size-in-bytes = ${?ACL_MIN_FILE_SIZE_IN_BYTES}
+  removal-cutoff-millis = 60000
```

Comment:
Totally up to you, but we could start specifying these things in terms of minutes or hours and using `FiniteDuration`s instead. 60000ms is 1 minute right?